sky/clouds/aws.py

```diff
@@ -116,7 +116,7 @@ class AWS(clouds.Cloud):
     @classmethod
     def _cloud_unsupported_features(
             cls) -> Dict[clouds.CloudImplementationFeatures, str]:
-        return {}
+        return dict()
```

Comment:
what's the difference here

Reply:
It's a remnant from a previous change. : )